sssoja/tv-bland
# moves UI components to subdirectory

### What changes have you made?
- Created a subdirectory called `styled` to separate functional from UI components 
- Updated the pathnames of all `Grid`, `Flex` and `Padding` imports

### Which issue(s) does this PR fix?
Fixes #12 

### Screenshots (if there are design changes)
No design changes

### How to test

Check for a subdirectory in the components folder called `styled` with the following UI components: 
- `Grid`
- `Flex`
- `Padding`

When you open `localhost:3000` the updated pathnames should not throw any errors
